tensorflow/inference/docker/2.5/py3/cu112/Dockerfile.gpu

```diff
@@ -91,10 +94,14 @@ RUN ${PIP} --no-cache-dir install --upgrade \
 # We don't install libnvinfer-dev since we don't need to build against TensorRT
 # cuda-x.x is CUDA version 10.2, 11.0, or 11.1 (for 11.2)
 # https://docs.nvidia.com/deeplearning/tensorrt/archives/tensorrt-722/install-guide/index.html
-RUN apt-get update \
- && apt-get install -y --no-install-recommends --allow-unauthenticated  \
-    libnvinfer7=${TF_TENSORRT_VERSION}-1+cuda11.1 \
-    libnvinfer-plugin7=${TF_TENSORRT_VERSION}-1+cuda11.1 \
+# https://forums.developer.nvidia.com/t/notice-cuda-linux-repository-key-rotation/212771
```

Comment:
can we use the latest version of tf serving api and model server? 2.5.4 I believe - https://pypi.org/project/tensorflow-serving-api/2.5.4/

Reply:
Using the TFS2.5.1 as the rest of the patch releases are dummy